scripts/git_check_diff

```diff
@@ -5,4 +5,7 @@ readonly SCRIPTS_DIR="$(dirname "$0")"
 source "$SCRIPTS_DIR/common"
 
 # Check that any generated files match those that are checked in.
-git diff --exit-code -- . ':!*.bazelrc'
+if [[ $(git status --short | wc --line) -gt 0 ]]; then
+    # One or more files are either modified or newly generated, exit with an error code
```

Comment:
Could you make sure this actually prints the diff though? Otherwise if it fails we need to spend even more time trying to reproduce locally by rebuilding everything.

Reply:
Changed it to print the complete list of modified or added files.